api/client-server/administrative_contact.yaml

```
@@ -291,9 +344,38 @@ paths:
                 required: ['id_server']
       responses:
         200:
-          description: An SMS message was sent to the given phone number.
+          description: |-
+            An SMS message was sent to the given phone number.
+
+            ``submit_url`` is an optional field containing a URL where the
+            client must submit a validation token to, with identical parameters
+            to the Identity Service API's ``/validate/msisdn/submitToken``
+            endpoint. The homeserver will send this token to the user, which
+            should then be prompted to provide it to the client.
+
+            If this field is not present, the client can assume that
+            verification will happen without the client's involvement.
           schema:
-            $ref: "../identity/definitions/sid.yaml"
+            type: object
+            allOf:
+              - $ref: "../identity/definitions/sid.yaml"
+              - type: object
+                properties:
+                  submit_url:
+                    type: string
+                    description: |-
+                      An optional field containing a URL where the client
+                      must submit a validation token to, with identical
+                      parameters to the Identity Service API's
+                      ``/validate/email/submitToken`` endpoint. The homeserver
+                      will send this token to the user, which should then be
```

Do this elsewhere too please (I won't keep pointing them out):
```suggestion
                      must send this token to the user (if applicable), which should then be
```